analyzer/restapi-plugin/src/main/java/eu/fasten/analyzer/restapiplugin/mvn/GraphResolversInitializer.java

```diff
@@ -0,0 +1,31 @@
+package eu.fasten.analyzer.restapiplugin.mvn;
+
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+public class GraphResolversInitializer {
```

Comment:
This might be a silly question!
Why sending a request to initialize graph resolvers? These should be initialized asynchronously when the REST API server is started. Right?

Reply:
The problem is that the server normally only starts when all endpoints are initialized. Thsi means that startup takes 10min when the graph has to be built. If we make everything lazy, then the startup is instant, but the first request takes 10min. We decided to do a compromise and start lazy, but immediately place a request for the graph, so the graph building starts right away. It still takes 10min for the graph to be available after startup, but the rest of the API is already available in this time.